.github/workflows/CompatHelpter.yml

```
@@ -1,16 +1,45 @@
 name: CompatHelper
 on:
   schedule:
-    - cron: 23 23 * * *
+    - cron: 0 0 * * *
```

I don't like loading github's servers at midnight, when everyone else has their crons runnings. 
```suggestion
    - cron: 23 23 * * *
```